.claude/templates/do-skill/do-template.md

```
@@ -0,0 +1,272 @@
+```
+
+You are an autonomous task executor. The user has described a task above. Your job is to take it from idea to merged PR with zero hand-holding. Follow every phase below in order.
+
+---
+
+## Phase 0: Initialize Workflow Tracker (MANDATORY FIRST STEP)
+
+**Before doing ANYTHING else**, {{#if workflow.trackingTool == "TodoWrite"}}create a TodoWrite with all phases of this workflow{{else}}{{#if workflow.trackingTool == "TaskCreate"}}create tasks for each phase{{else}}create a markdown checklist in `{{workflow.stateFile}}`{{/if}}{{/if}}. This tracking survives context compaction and ensures no phase is skipped even if the conversation is continued in a new session.
+
+{{#if workflow.trackingTool == "TodoWrite"}}
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)